### PR TITLE
DTFS2-4545 - update portal mocks to use TFM bank

### PR DESCRIPTION
**Pull request by ttbarnes**
- remove "no real banks" mocks in mock-data-loader
- simplify mock users so we have only 2x TFM banks (e.g BANK1_MAKER1) and the admin/editor users
- update all portal E2E tests & fixtures to use the new logins and banks

### CODE REVIEW

File: e2e-tests/portal/cypress/integration/journeys/maker/reports/audit-supply-contracts/report-should-only-show-data-from-users-bank.spec.js [hunk 1]
```diff
@@ -29,27 +22,20 @@ context('Audit - Report', () => {
...
+    cy.deleteDeals(BANK1_MAKER);
+    cy.deleteDeals(BANK2_MAKER);
 
-    cy.insertManyDeals([aDealWithTenLoansAndTenBonds, aDealWithOneLoan], HSBC_MAKER_1)
-      .then((insertedDeals) => hsbcDeals.push(insertedDeals));
+    cy.insertManyDeals([aDealWithOneBond, aDealWithTenLoans], BANK1_MAKER);
 
-    cy.insertManyDeals([aDealWithOneLoanAndOneBond, aDealWithTenLoans], HSBC_MAKER_2)
-      .then((insertedDeals) => hsbcDeals.push(insertedDeals));
+    cy.insertManyDeals([aDealWithTenLoansAndTenBonds, aDealWithOneLoan], BANK2_MAKER);
```
Comment (pricea1) [inline e2e-tests/portal/cypress/integration/journeys/maker/reports/audit-supply-contracts/report-should-only-show-data-from-users-bank.spec.js:30]: We've lost the test where a maker of one bank can see deals made by a maker of same bank

**Review by pricea1 — COMMENTED** [review]
I think the test for viewing deals my maker on same bank has been lost.

File: e2e-tests/portal/cypress/integration/journeys/maker/reports/audit-supply-contracts/report-should-only-show-data-from-users-bank.spec.js (hunk 1 above)
Comment (ttbarnes) [reply e2e-tests/portal/cypress/integration/journeys/maker/reports/audit-supply-contracts/report-should-only-show-data-from-users-bank.spec.js:39]: Good spot, i've updated the test here: https://github.com/foundry4/dtfs2/pull/197/commits/43da29f634f1937e6e206c83cecdffa75cc68f9d

**Review by pricea1 — APPROVED** [review]
👍

**Pull request merged**